src/aat/resources/features/F-027/F-027.feature

```diff
@@ -0,0 +1,58 @@
+@F-027
+Feature: Fetch a case for display for Case Worker
+
+  Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+
+  @S-038
+  Scenario: must return status 200 along with the case-view object successfully
+    Given a user with [an active profile in CCD]
+    And a case that has just been created as in [Standard_Full_Case]
+    When a request is prepared with appropriate values
+    And the request [contains as input a valid case-reference that exists in CCD]
+    And it is submitted to call the [Fetch a case for display for Case Worker] operation of [CCD Data Store]
+    Then a positive response is received
+    And the response [has the 200 return code]
+    And the response has all other details as expected
+
+  @S-035
+  Scenario: must return appropriate negative response when case id is structurally valid but not exist in CCD
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [contains in input parameters a structurally valid but non-existing case-reference]
+    And it is submitted to call the [Fetch a case for display for Case Worker] operation of [CCD Data Store]
+    Then a negative response is received
+    And the response [has the 400 return code]
+    And the response has all other details as expected
+
+  @S-036
+  Scenario: must return appropriate negative response when request does not provide valid authentication credentials
+    Given a user with [an active profile in CCD]
+    And a case that has just been created as in [Standard_Full_Case]
```

Comment:
why create a case before a call known to be going to fail? That is unnecessary and confusing.

Reply:
It is kept for the purpose that expect the scenario related spec rest everything is correct and correct case reference is also part of this otherwise that spec and prepared data will be misleading.

Reply:
It is kept for the purpose that request contains everything apart from the specified scenario otherwise prepared data and spec would be misleading. however I can remove that if you want.

Reply:
case creation removed.